backend/de.metas.fresh/de.metas.fresh.base/src/main/jasperreports/de/metas/docs/label/QR_Label_badge.jrxml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Created with Jaspersoft Studio version 6.5.1.final using JasperReports Library version 6.5.1  -->
+<jasperReport xmlns="http://jasperreports.sourceforge.net/jasperreports" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://jasperreports.sourceforge.net/jasperreports http://jasperreports.sourceforge.net/xsd/jasperreport.xsd" name="QR Label" printOrder="Horizontal" pageWidth="70" pageHeight="70" columnWidth="70" leftMargin="0" rightMargin="0" topMargin="0" bottomMargin="0" uuid="d77f60d8-3ca8-42d0-9ca2-d19df969e682">
```

Comment:
pls move this file to `de/metas/security` new directory (instead of `de/metas/docs/label`) and rename it to `AuthBadge_2cm.jrxml`.

@cristinamghita  wdyt?

Reply:
done

Reply:
> pls move this file to `de/metas/security` new directory (instead of `de/metas/docs/label`) and rename it to `AuthBadge_2cm.jrxml`.
> 
> @cristinamghita wdyt?

Agree is not like a regular label. Has other meaning.